src/components/login/LoginForm.scss

```diff
@@ -1,36 +1,48 @@
-input {
-    width: 100%;
-}
+.login-container {
+    font-family: 'Montserrat', sans-serif;
+    margin: 30%;
 
-.label {
-    color: white;
-}
+    h1 {
+        font-size: 2.5em;
+        font-weight: 800;
+    }
 
-.button {
-    width: 100%;
-    color: white;
-}
+    input {
+        width: 100%;
+    }
 
-.button:hover {
-    cursor: pointer;
-    box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-}
+    label {
+        color: #fff;
+    }
 
-.login-button {
-    background: #ecb8b8;
-}
+    .button {
+        color: #fff;
+        font-family: 'Montserrat', sans-serif;
+        font-weight: 800;
+        width: 100%;
+    }
+
+    .button:hover {
+        box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+        cursor: pointer;
+    }
+
+    .login-button {
+        background: #ecb8b8;
+    }
```

Comment:
Point of interest - uses bold Montserrat font in most places according to Figma spec